components/Alert/react/ReadMe.md

```diff
@@ -0,0 +1,83 @@
+# Alert
```

Comment:
All the details in this README are great. 🤩 It would be nice to have those all reflected in Storybook, whether manually or programatically.